public/index.html

```diff
@@ -1,8 +1,11 @@
 <!doctype html>
 <html>
+    <meta http-equiv="Content-Security-Policy" content="default-src https://cdn.example.net; child-src 'none'; object-src 'none'">
```

Comment:
What is this tag used for?

Reply:
I was getting some error related to the content-security-policy and this is what people suggested. But after all I figured that CORS solve this, I just checked and looks like without this meta tag is everything fine as well. Will delete it now.